src/ArgentPonyWarcraftClient/Models/GameDataApi/ModifiedCrafting/ModifiedCraftingReagentSlotTypeIndex.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace ArgentPonyWarcraftClient
+{
+    /// <summary>
+    /// An index of Modified Crafting reagent slot types.
+    /// </summary>
+    public class ModifiedCraftingReagentSlotTypeIndex
+    {
+        /// <summary>
+        /// Gets links for the index of Modified Crafting reagent slot types.
+        /// </summary>
+        [JsonPropertyName("_links")]
+        public Links Links { get; set; }
+    }
```

Comment:
Looks like this is missing a `public SlotTypes { get; set; }` property
![image](https://user-images.githubusercontent.com/8053660/100098753-56c5ec00-2e13-11eb-92ab-387be5b63724.png)

Reply:
I thought I had added that property, but clearly not.  Fixed now.  Good catch!